### ISSUE TITLE
Tracheal bronchus

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6628]
**Preferred term label:**  Tracheal bronchus
**Synonyms**

**Definition (free text, please give PubMed ID)**
Aberrant or accessory bronchus supplying the upper lobe originating from the lateral wall of the trachea. The tracheal bronchus is more commonly right-sided, has a variable length and may be blind-ended. Two common types of tracheal bronchus are: supernumerary and displaced. PMID: 11158647, PMID: 19332762

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology HP:0025426

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
